src/editors/autocompleteEditor.js

```diff
@@ -77,247 +41,329 @@ function onBeforeKeyDown(event) {
   }
 }
 
-AutocompleteEditor.prototype.prepare = function(...args) {
-  HandsontableEditor.prototype.prepare.apply(this, args);
-};
+class AutocompleteEditor extends HandsontableEditor {
```

Comment:
It would be great, if you could add the JSDoc annotations to this class and all of its methods and properties. They should contain a short description of the method and its arguments. We're already doing it in all of the plugins, for example, so you can [use one of them as a reference](https://github.com/handsontable/handsontable/blob/develop/src/plugins/contextMenu/contextMenu.js).